tools/src/test/java/com/nedap/archie/adlparser/DateTimeParserTest.java

```diff
@@ -29,22 +29,42 @@ public void dateCorrect() {
     @Test
     public void datetimeCorrect() {
         assertEquals(OffsetDateTime.of(2015, 1, 1, 12, 1, 1, 100000000, ZoneOffset.of("+0100")), DateTimeParsers.parseDateTimeValue("2015-1-1T12:01:01,1+0100"));
-        assertEquals(OffsetDateTime.of(2015, 1, 1, 12, 1, 1, 0, ZoneOffset.of("+0100")), DateTimeParsers.parseDateTimeValue("2015-1-1T12:01:01+0100"));
+        assertEquals(OffsetDateTime.of(2015, 1, 1, 12, 1, 1, 100000000, ZoneOffset.of("+0100")), DateTimeParsers.parseDateTimeValue("2015-1-1T12:01:01,1+0100"));
```

Comment:
Can the previous string still be parsed now? (`2015-1-1T12:01:01+0100`) Could you put that example back if so?

Reply:
re-added it